pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/JavaInstanceMain.java

```diff
@@ -221,5 +222,22 @@ public void getFunctionStatus(Empty request, StreamObserver<InstanceCommunicatio
                 throw new RuntimeException(e);
             }
         }
+
```

Comment:
arent we removing metrics collection from spawner?

Reply:
Sure but I would say lets do it in multiple PRs